extends Node2D

@export var nextlevel = ""

func _ready() -> void:
	Globals.set_music(load("res://kaynak/ses/Bolum2.mp3"))
	if Globals.otk:
		$Panel/Label.text = \
		"ð°€ð°£ð°‘ð°†ð°žð°–ð°€:ð°ð°‡ð°²ð°˜ð°†ð°ºð°”:ð°²ð°€ð°‹ð°“ð°€ð±… " +\
		"ð°¶ð°žð°€ð°º:ð°‰ð°€ð°“ð°ƒð°¾ð°¤ð°€:ð°†ð°žð°€ð±ð°£:ð°´ð°€ð°‘ð°º: ð°€ð°¯ð°ƒð° ð°€ð°˜ð°ƒ:ð°¸ð°†ð°ºð°¢ð°€ð°£:ð°ð°€ð°¼ð°š " +\
		"ð°ð°€ð° ð°¤:ð°“ð°‡ð±ð°¢ð°€ð°£ð°žð°€ð°º:ð°´ð°€ð°ºð±ð°ƒ:ð°€ð°–ð°¶:ð°†ð°ž "
		
		$devam.text = "ð°“ð°€ð°‰ð°¢:ð°€ð±…ð°¢ð°€ð°š:ð°ƒð°²ð°¤:E:ð±ƒð°†ð±ð°£ð°€:ð°‰ð°€ð°½"
	else:
		$Panel/Label.text =  \
		"Anadoluya gÃ¶Ã§Ã¼yoruz Cevdet! " + \
		"Ihlara Vadisi'ne ulaÅŸana kadar kafileyi koruman gerek. " + \
		"Gelen dÃ¼ÅŸmanlara karÅŸÄ± ayÄ±k ol"
		$devam.text = "Devam etmek iÃ§in E tuÅŸuna bas"

func _process(delta: float) -> void:
	if Input.is_action_just_pressed("ui_attack"):
		get_tree().change_scene_to_file(nextlevel)
